### DISCUSSION TITLE
[V5] Change the default format of logger to json

### DISCUSSION THREAD

**Discussion by 3rownz** [adonisjs/core#2507] [Help] [answered]
Im trying to get the format of the adonis 5 logger to ouput similar to the default pino json format. Has anyone got a solution of how to achieve this?

**Comment by thetutlage** [comment, marked as answer]
I think it has the same format. During development we pretty print the logs using [pino pretty](https://github.com/pinojs/pino-pretty). You can disable pretty printing by editing the logger config inside the `config/app.ts` file

**Reply by 3rownz** [reply]
Thanks this did the job! The default pino allows mixins ...looks like the adonis logger doesn't, I need to use this for including a variable (like a request url). Although I can add a static string via a child. Will this be added or am I missing something?
